docs/sources/visualizations/histogram-draft.md

```diff
@@ -0,0 +1,107 @@
+| `approximation using PromQL`  |   `histogram_quantile(0.75, uploaded_image_bytes_bucket)`                                             |       ratio, buckets       | The function takes a ratio and the histogram’s buckets as input and returns an approximation of the value at the point of the ratio’s quantile. |
+
+You can use one of the following **panels** to visualize their query:
+
+- **Stat.** Shows the summary of a single series; displays only one metric per panel
+- **Gauge.** Also known as a speedometer; best used when the data had defined boundary limits to warn users when they are falling under the normal range
+- **Graph.** Time charts that display data points over a time axis allowing users to overlap metrics to compare them overtime; easily modified and good for tracking outliers, state changes, or triggers
+
+This sample query is best represented using a stat if users want to see files they currently have as opposed to over time (which is better shown as a graph). A gauge would not be feasible without a defined range.
+
+## Using PromQL to approximate
+
+We cannot get an accurate value of what size is a quarter of the files smaller than, but we can get an approximation by using PromQL’s `histogram_quantile` function.
+
+The function returns an approximation somewhere in the correct bucket based on our knowledge of exactly how many values are above a particular bucket and how many values are below it.
+
+If the approximated value is larger than the largest bucket (excluding the `+Inf` bucket), Prometheus will give up and give you the value of the largest bucket’s `le` back.
+
+We can then make our approximation of the third quartile with the following query:
+
+```bash
+histogram_quantile(0.75, uploaded_image_bytes_bucket)
+```
+
+**Note:** When talking about service level, the precision of quantile estimations is relevant. Historically, a lot of services are defined as something like “the p95 latency may not exceed 0.25 seconds.” Assuming we have a bucket for `le=0.25`, we *can* accurately answer whether or not the p95 latency does exceed 0.25 or not.
+
+However, since the p95 value is approximated, we cannot tell definitively if p95 is, say, 0.22 or 0.24 without a bucket in between the two. A way of phrasing this requirement so we do get an accurate number of how close we are to violating our service level is “the proportion of requests in which latency exceeds 0.25 seconds must be less than 5 percent.” Instead of approximating the p95 and seeing if it’s below or above 0.25 seconds, we precisely define the percentage of requests exceeding 0.25 seconds using the methods from above.
```

Comment:
```suggestion
However, since the p95 value is approximated, we cannot definitively identify if p95 is, for example, 0.22 or 0.24 without a bucket between the two. A way of phrasing this requirement so we have an accurate potential service level violations is “the proportion of requests in which latency exceeds 0.25 seconds must be less than 5 percent.” Instead of approximating the p95 and seeing if it’s below or above 0.25 seconds, we precisely define the percentage of requests exceeding 0.25 seconds using the methods from above.
```